src/becca/entities/bnote.ts

```diff
@@ -78,13 +78,13 @@ class BNote extends AbstractBeccaEntity<BNote> {
 
     // following attributes are filled during searching in the database
     /** size of the content in bytes */
-    private contentSize!: number | null;
+    contentSize!: number | null;
```

Comment:
What's the reason for making these fields public?

Reply:
When I added the fields to `BNote`, I made them `private` implicitly.
The value is set externally in `services/search/services/search.ts` so it had to be made `public`.